src/components/shared/passphraseRenderer/index.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import fillWordsList from 'bitcore-mnemonic/lib/words/english';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { withTranslation } from 'react-i18next';
+import styles from './passphraseRenderer.css';
+import { PrimaryButton, TertiaryButton } from '../../toolbox/buttons/button';
+
+class PassphraseRenderer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      indexes: [2, 9],
+      fieldSelected: undefined,
+      displayOptions: undefined,
+      chosenWords: {},
+      options: {},
+      isCorrect: false,
+      hasErrors: false,
+      disabledButton: true,
+    };
+
+    this.handleConfirm = this.handleConfirm.bind(this);
+  }
+
+  UNSAFE_componentWillMount() { // eslint-disable-line camelcase
```

Comment:
💅 IMO logic in this method should be moved to `constructor` since `UNSAFE_componentWillMount` is deprecated.

Reply:
You're right. Although from what I see there's no reason it shouldn't work inside `componentDidMount` instead of `componentWillMount`. I moved it there, what do you think?

Reply:
I think it's less code and more clear to the reader if it's all in one space the `constructor`.